ignite/services/plugin/plugin.go

```diff
@@ -148,18 +148,8 @@ func newPlugin(pluginsDir string, cp pluginsconfig.Plugin, options ...Option) *P
 		apply(p)
 	}
 
-	// This is a local plugin, check if the file exists
-	if pluginsconfig.IsLocalPath(pluginPath) {
-		// if directory is relative, make it absolute
-		if !filepath.IsAbs(pluginPath) {
-			pluginPathAbs, err := filepath.Abs(pluginPath)
-			if err != nil {
-				p.Error = errors.Errorf("failed to get absolute path of %s: %w", pluginPath, err)
-				return p
-			}
-
-			pluginPath = pluginPathAbs
-		}
+	// This is a local plugin, check if the directory exists
+	if filepath.IsAbs(pluginPath) {
```

Comment:
lgtm, but, what if you already have relative paths in your igniteapp.yml in your chain locally and in an app.

Maybe `ignite doctor` should do something about it (or simply fail if that's the case)

Reply:
We are already using only the absolute path here. We filter at the top level of the application. If it is a relative path, we convert it to an absolute path. In this part, the path is already absolute